### ISSUE TITLE
/missing-playwright-await > silent rule when using it in Promises

### ISSUE THREAD

**Issue by Juanicanete10** [playwright-community/eslint-plugin-playwright#107]
Having the following block produce a missing-playwright-await error 

![image](https://user-images.githubusercontent.com/87318561/196512808-80dc7581-7dbf-4e11-b77c-d00e7908fe10.png)

```
		await Promise.all([
			expect(previewLessonPage.relatedContent.title).toBeHidden(),
			expect(previewLessonPage.relatedContent.items).toHaveCount(0),
		]);
```

Those cases should not produce errors considering that we're handling them into a Promise already and that promise it's being awaited.

**Comment by mskelton** [comment]
What's the use case?  Your example could just as easily be written like so:

```js
await expect(previewLessonPage.relatedContent.title).toBeHidden()
await expect(previewLessonPage.relatedContent.items).toHaveCount(0)
```

**Comment by mskelton** [comment]
Interesting, that seems like a really niche use case, but adding support for promise.all shouldn't be to difficult.
